### ISSUE TITLE
`@stylistic/linebreaks` drops every comment a value holds

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#235]
Found by the comment oracle written for the plan of 2026-08-20, which fixes every fixture with every rule under every primary option and counts the comments that came out against the comments that went in.

The same cause as the `unit-case` comment loss filed beside this one, in the one rule that rewrites the whole document.

### What minimal example or steps are needed to reproduce the bug?

A comment inside a multi-line value:

```css
a { b: 1px /* c */
	2px; }
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "rules": {
    "@stylistic/linebreaks": "windows"
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.css" --config myconfig.json --fix
```

### What did you expect to happen?

Every line ending rewritten, and every other character of the file left exactly as it was.

### What actually happened?

```css
a { b: 1px
	2px; }
```

The comment is gone, and nothing is reported. Plain CSS, `postcss-scss` and `postcss-less` all go the same way, and under `postcss-scss` an inline comment is lost too: `a { b: 1px // c\n\t2px; }` comes back as `a { b: 1px \r\n\t2px; }`.

A comment standing as a node of its own survives; only the ones a value or a set of parameters holds are lost.

### What is the cause?

The rule rewrites every value it walks, and the copy it rewrites is the cleaned one PostCSS hands back from `decl.value`, not the raw the comments live in. Writing the bare property discards `raws.value`, and with it the comment and, under `postcss-scss`, the copy that syntax prints.

`getDeclarationValue` and `setDeclarationValue` are the pair that answers this, and the rule calls neither.
